Cargo.toml

```diff
@@ -3,9 +3,11 @@ members = [
     "crates/metrics",
     "crates/witness-generator",
     "crates/witness-generator-cli",
-    "xtask",
-    # Ere host crate
     "crates/ere-hosts",
+
+    "ere-guests/libs",
```

Comment:
Starting to prepare the stage for the shared lib to be used for guests and required stdin types.